optimum/intel/openvino/utils.py

```diff
@@ -123,3 +126,19 @@ def _is_timm_ov_dir(model_dir):
         if hf_hub_id and model_info(hf_hub_id).library_name == "timm":
             return True
     return False
+
+
+def print_compiled_model_properties(compiled_model):
```

Comment:
```suggestion
def _print_compiled_model_properties(compiled_model):
```

to leave us more freedom in the future in case we want to add major changes or even deprecate